traveler-ios-ui/TravelerKitUI/Views/ProductCancelCell.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  ProductCancelCell.swift
+//  TravelerKitUI
+//
+//  Created by Omar Padierna on 2019-06-04.
+//  Copyright © 2019 GuestLogix. All rights reserved.
+//
+
+import UIKit
+
+protocol ProductCancelCellDelegate: class {
+    func productCancelDidPressButton (_ cell: ProductCancelCell)
+}
+
+class ProductCancelCell: UITableViewCell {
```

Comment:
Views should not have knowledge of its use case. This one here is speaking Product and Cancel.

Use more generic terms

Reply:
All this is, is a cell with three labels. Thats all it should know